drv/transceivers-server/Cargo.toml

```diff
@@ -33,6 +33,7 @@ zerocopy = { workspace = true }
 
 [features]
 vlan = ["task-net-api/vlan"]
+thermal-control = []
```

Comment:
would it make sense to make the dependency on `task-thermal-api` optional?

Reply:
It certainly could!